cmd/unifi_exporter/main.go

```diff
@@ -5,53 +5,79 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/mdlayher/unifi"
```

Comment:
I think goimports does this by default, but mind moving all third party imports into their own block? e.g.

```go
import (
    "io/ioutil"
    "net"

    "github.com/mdlayher/unifi"
    "gopkg.in/yaml.v2"
)
```